docs/proposals/PROPOSAL_E.md

```diff
@@ -94,7 +94,7 @@ Extend the Image Manifest with a refers field (existing registries should ignore
 }
 ```
 
-### Registry HTTP API
+### Distribution Spec
```

Comment:
this is changing of the template format, but at this point I think its ok

Reply:
Oh ok. That wasn't my intention. I think we can ignore this comment then.